stories/Input.stories.ts

```diff
@@ -0,0 +1,23 @@
+// YourComponent.stories.ts|tsx
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { InputDemo } from "./Input.demo";
+
+//👇 This default export determines where your story goes in the story list
+const meta: Meta<typeof InputDemo> = {
+  component: InputDemo,
```

Comment:
I think it'd be useful to add a `title` attribute here like in the Accordion story, it's how Storybook determines where to place the page in the sidebar and can use it to place the component within a category.

Reply:
We can also add the `tags: ['autodocs'],` option and Storybook will autogenerate a docs page for us.